blockmanager.go

```diff
@@ -109,6 +109,9 @@ type blockManagerCfg struct {
 		checkResponse func(sp *ServerPeer, resp wire.Message,
 			quit chan<- struct{}, peerQuit chan<- struct{}),
 		options ...QueryOption)
+
+	// blkHdrSideLoad is the config required for block header side loading.
+	blkHdrSideLoad *SideLoadOpt
```

Comment:
I think we can decouple this further. This can live in the main server initialization, before the block manager has been fully started.

Reply:
https://github.com/lightninglabs/neutrino/pull/285#discussion_r1414517226